src/components/ErrorComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { translate } from '../plugins/i18n'
+
+const reload = () => {
+  window.location.reload()
+}
+
+const Oops = ({ t, errorType }) => (
```

Comment:
Yes that was I was afraid of. What about  `ErrorPanel` or `ErrorMessage` or `Failure` ?